src/Field/index.js

```diff
@@ -3,15 +3,22 @@ import {default as PT} from 'prop-types'
 import cx from 'classnames'
 
 const Field = ({ id, label, hint, error, description, disabled, children }) => {
+  
+  let elements = React.Children.toArray(children)
   return (
     <div className={cx('ola_field', {'is-invalid': error}, {'is-disabled': disabled})}>
       <label htmlFor={id} className="ola_field-label">
         { label }
         {hint && <span className="ola_field-hint">{ hint }</span>}
       </label>
       <div className="ola_field-input">
-        {React.cloneElement(children, { id: id, error, disabled })}
+        {React.cloneElement(elements[0], { id: id, error, disabled })}
       </div>
+      {elements.map((item,idx) => {
+        if (idx >= 1) {
+          return item
+        }
+      })}
       {description && <p className={ cx({ 'ola_field-error': error, 'ola_field-description': !error }) }>{description}</p>}
```

Comment:
Hi, you can use `shift()` for a more concise way to achieve this:

```js
<div className="ola_field-input">
    {React.cloneElement(elements.shift(), { id: id, error, disabled })}
   {elements}
</div>
```

Reply:
Yep